packages/victory-core/src/victory-animation/victory-animation.js

```diff
@@ -66,9 +67,7 @@ export default class VictoryAnimation extends React.Component {
     easing: "quadInOut"
   };
 
-  static contextTypes = {
-    getTimer: PropTypes.func
-  };
+  static contextType = TimerContext;
 
   constructor(props) {
```

Comment:
I found that React requires you to pass the `context` constructor argument to the `super` call in order for `this.context` to be set. Right now, `this.context` is always undefined.